rn/AuthProvider.js

```diff
@@ -21,7 +21,7 @@ const AuthProvider = ({children}) => {
     const creds = Realm.Credentials.emailPassword(email, password);
```

Comment:
Looks like some extra commits snuck in here. We'll have to do some light git surgery before merge.